src/downloader.cpp

```diff
@@ -93,6 +93,22 @@ void Download::updateStatus(bool follow)
   }
 }
 
+void Download::resumeDownload()
+{
```

Comment:
I'm not sure but we may need to update the status here.
Either before, to be sure that the `m_followedBy` is correctly set.
Or after, to be sure that the status is updated after the call.